src/audio/mod.rs

```diff
@@ -1,39 +1,53 @@
-use crate::{ring_buffer, Pipeline, Result};
+pub mod midi;
+
+use crate::{ring_buffer, AudioUnit, Pipeline, Result};
 use anyhow::anyhow;
 use cpal::{
     traits::{DeviceTrait, HostTrait, StreamTrait},
     Device, StreamConfig,
 };
 use ringbuf::RingBuffer;
 use std::time::Duration;
+use wmidi::MidiMessage;
 
-const LATENCY: f32 = 20.0;
+const LATENCY: f32 = 50.0;
```

Comment:
Turned this up a little since 20 is too low on my linux machine. This should probably go in the config file.